src/main/java/org/openmainframeproject/cobolcheck/features/testSuiteParser/TestSuiteParser.java

```diff
@@ -794,7 +794,6 @@ private void handleEndOfMockStatement(BufferedReader testSuiteReader, String tes
      *
      * @param parsedTestSuiteLines The parsed lines, that the generated lines are
      *                             appended to
-     * @return - the next token from the testSuiteReader.
```

Comment:
Please remove these changes, as we do not want to invest in javadoc at this moment.